src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
```

Comment:
Couldn't we just include `port` inside `host`? For example, we could specify `host` like this: `airbrake.io:443`.